src/ForgottenPassword/index.js

```diff
@@ -132,14 +132,20 @@ class ForgottenPassword extends React.Component<Props, State> {
     this.setState({ email: e.target.value });
   };
 
-  handleSubmitEmail = (e: SyntheticEvent<HTMLFormElement>) => {
+  handleSubmitEmail = async (e: SyntheticEvent<HTMLFormElement>) => {
     e.preventDefault();
     const { email } = this.state;
-    Requester.resetPassword(email);
-    this.props.history.push({
-      pathname: '/check-recovery-link',
-      state: { email },
-    });
+    try {
+      await resetPassword(email);
+
+      this.props.history.push({
+        pathname: '/check-recovery-link',
+        state: { email },
+      });
+    } catch (e) {
+      //
```

Comment:
Looks good! Just a little comment here so we know whats happening 🙂

Reply:
Oh sorry, I've forgotten. :)